### PR TITLE
T/125: Use a more secure way to access GitHub on Travis

### LINKED ISSUE #125: Use a more secure way to access GitHub on Travis
We've been doing `git clone GH_USER:GH_PASSWORD@github.com/....` which meant that if something goes wrong we'd leak the password to Travis logs.

Let's switch to use tokens and secure the commands so errors goes to dev/null.

### PR DESCRIPTION
**Pull request by pomek**
Fix: Script for saving revision was not safe. Closes #125.

If git failed, the credentials data was printed to the public logs.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/bin/save-revision.js [hunk 1]
```diff
@@ -19,11 +19,10 @@ const { tools } = require( '@ckeditor/ckeditor5-dev-utils' );
...
 const revisionBranch = `${ branch }-revisions`;
 
+exec( `echo "https://${ process.env.GITHUB_TOKEN }:@github.com" > .git/credentials 2> /dev/null` );
+
 // Clone the repository.
-exec(
-	`git clone -b ${ revisionBranch } ` +
-	`https://${ process.env.GITHUB_USER }:${ process.env.GITHUB_PASSWORD }@github.com/ckeditor/ckeditor5.git`
-);
+exec( `git clone -b ${ revisionBranch } https://github.com/ckeditor/ckeditor5.git` );
```
Comment (Reinmar) [inline packages/ckeditor5-dev-tests/bin/save-revision.js]: Why quite push if clone isn't quite?

Comment (pomek) [inline packages/ckeditor5-dev-tests/bin/save-revision.js]: Fixed.

**Comment by Reinmar** [comment]
Let's see if it works. I'll merge this PR and check it on Travis.

**Comment by Reinmar** [comment]
OK, It didn't work: https://travis-ci.org/ckeditor/ckeditor5-engine/builds/211033951#L1473. I'll check it manually.

**Comment by Reinmar** [comment]
I think it should be this way: https://github.com/ckeditor/ckeditor5-dev/commit/308c473e293066a5874b6ebe74f3da963e1cff06

It worked locally.

**Comment by Reinmar** [comment]
🎉  https://github.com/ckeditor/ckeditor5/commit/fc516a0cf8733de6b4ebb229088d0a68bc41c635

**Pull request merged**